tensorflow/compiler/tests/xla_test.py

```diff
@@ -44,18 +44,28 @@
 flags.DEFINE_string('types', None, 'Types to test. Comma-separated list.')
 flags.DEFINE_string('disabled_manifest', None,
                     'Path to a file with a list of tests that should not run.')
+flags.DEFINE_string('load_plugin', None,
+                    'Load plugin details from python script.')
 
 
 class XLATestCase(test.TestCase):
   """XLA test cases are parameterized test cases."""
 
   def __init__(self, method_name='runTest'):
     super(XLATestCase, self).__init__(method_name)
-    self.device = FLAGS.test_device
+    test_device = FLAGS.test_device
+    test_types = FLAGS.types
+
+    if FLAGS.load_plugin:
```

Comment:
I think we could simplify this change. The design here is that we already take the type and device inputs via flags passed from the BUILD file. I don't think we need to do anything different for the plugin case --- we can still pass these parameters via flags; all the magic can live in the build file.

I think you can revert the changes to xla_test.py and remove plugin_config.tpl.